docs/source/specification.md

```diff
@@ -66,6 +66,7 @@ In addition, we also describe some {ref}`roles <satre_roles>` that are necessary
 The SATRE specification has been developed based on the following principles:
 
 - TREs should be as easy as possible for end-users to use (_e.g._ researchers) whilst still remaining secure.
+- TREs should take accessibility for all users with disabilities into account
```

Comment:
```suggestion
- TREs should take accessibility for all users, including those with disabilities, into account.
```

Accessibility matters for all users. I suppose there is an argument that, for example, only providing a command-line interface for medical professionals is poor accessibility.

Do we want to say it is _particularly_ important to consider users with disabilities?

There might be TPS people at the Turing to would know how to better phrase this.